frontend/src/pages/ActivityReport/Pages/Review/Approver/Approved.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Approved = ({
+  additionalNotes,
+  managerNotes,
+}) => (
+  <>
+    <h2>Report approved</h2>
+    <div className="smart-hub--creator-notes">
+      <p>
+        <span className="text-bold">Creator notes</span>
+        <br />
+        <br />
+        { additionalNotes }
+      </p>
+    </div>
+    <div className="smart-hub--creator-notes margin-top-2">
+      <p>
+        <span className="text-bold">Manager notes</span>
+        <br />
+        <br />
+        { managerNotes }
```

Comment:
This shows the old manager notes if I saved some along with `needs action` and then changed them when I switched to `approved`